bin/license_functions.js

```diff
@@ -15,18 +15,26 @@ module.exports.dispatcher = function (args) {
 
     switch (args[0].toLowerCase()) {
         case 'install': {
-            let argsWithoutFilename = args.splice(1, args.length - 2);
+            let argsWithoutFilename;
+            let shouldInstallSlm = isSlm(args);
+            if (shouldInstallSlm) {
+                argsWithoutFilename = args.splice(1, args.length - 1);
```

Comment:
Better practice to use slice() instead of splice() for array manipulation.
Splice() changes the content of the args array, which is passed as function parameter. This can have unexpected results, as the args parameter is passed by reference.
Slice() lets you access part of an array without modifying it.